yaml/com/ibm/VPD/Manager.interface.yaml

```diff
@@ -94,45 +94,12 @@ methods:
                 Location code in expanded format.
       errors:
           - xyz.openbmc_project.Common.Error.InvalidArgument
-          - com.ibm.VPD.Error.LocationNotFound
-          - com.ibm.VPD.Error.NodeNotFound
+          - com.ibm.VPD.VPD.Error.LocationNotFound
```

Comment:
Lets see if we can rename the file from VPD.errors.yaml. 
These repeated VPD.VPD is not looking good and create confusions later on.

Reply:
Something like Parser.errors.yaml

Reply:
okay, yeah.
but then it will be different from current upstream commit.